src/main/java/wooteco/subway/auth/infrastructure/JwtTokenProvider.java

```diff
@@ -1,42 +1,43 @@
 package wooteco.subway.auth.infrastructure;
 
 import io.jsonwebtoken.*;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.stereotype.Component;
 
 import java.util.Date;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import wooteco.subway.auth.application.AuthorizationException;
 
 @Component
 public class JwtTokenProvider {
+
     @Value("${security.jwt.token.secret-key}")
     private String secretKey;
     @Value("${security.jwt.token.expire-length}")
     private long validityInMilliseconds;
 
-    public String createToken(String payload) {
-        Claims claims = Jwts.claims().setSubject(payload);
+    public String createToken(Long id) {
         Date now = new Date();
         Date validity = new Date(now.getTime() + validityInMilliseconds);
 
         return Jwts.builder()
-                .setClaims(claims)
-                .setIssuedAt(now)
-                .setExpiration(validity)
-                .signWith(SignatureAlgorithm.HS256, secretKey)
-                .compact();
+            .claim("id", id)
+            .setIssuedAt(now)
+            .setExpiration(validity)
+            .signWith(SignatureAlgorithm.HS256, secretKey)
+            .compact();
     }
 
-    public String getPayload(String token) {
-        return Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token).getBody().getSubject();
+    public Long getIdFromPayLoad(String token) {
+        final Claims claims = Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token)
+            .getBody();
+        return claims.get("id", Long.class);
     }
 
-    public boolean validateToken(String token) {
+    public void validateToken(String token) {
         try {
             Jws<Claims> claims = Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token);
```

Comment:
사용하지 않는다면 변수로 받을 필요 없을 것 같아요 :) 
```suggestion
            Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token);
```

Reply:
제거 하였습니다!